corehq/apps/app_manager/models.py

```diff
@@ -1457,7 +1457,8 @@ def audio_by_language(self, lang, strict=False):
     def custom_icon_form_and_text_by_language(self, lang):
         custom_icon = self.custom_icon
         if custom_icon:
-            return custom_icon.form, custom_icon.text[lang]
+            custom_icon_text = custom_icon.text.get(lang) or custom_icon.text.get(self.get_app().default_language)
```

Comment:
Is `""` valid icon text?  If so, this could be potentially confusing, as the empty string is falsy.

Reply:
looks like the change `custom_icon.text.get(lang, custom_icon.text.get(self.get_app().default_language))` would take care of it. Will make that change.